pkg/build/gobuild.go

```diff
@@ -444,7 +461,7 @@ func (gb *gobuild) Build(ctx context.Context, s string) (v1.Image, error) {
 	}
 
 	// Do the build into a temporary file.
-	file, err := gb.build(ctx, ref.Path(), platform, gb.disableOptimizations)
+	file, err := gb.build(ctx, pkg.Dir, platform, gb.disableOptimizations)
```

Comment:
curious if this triggers the determination to fail

Reply:
switched to import path

Reply:
switch back to pkg dir and changed the test

Reply:
I'm guessing this is needed because it piggybacks on the normalization being done in importPackage, but this is the part of the change that makes me most nervous 🤔

Reply:
this is passing the absolute path to the package now